packages/material/scss/pager/_layout.scss

```diff
@@ -0,0 +1,29 @@
+$pager-buttons-size: 36px !default;
+
+@import "~@progress/kendo-theme-default/scss/pager/layout";
+
+@include exports ('pager/layout/material') {
+    $pager-elements-margin: 8px !default;
+    $pager-elements-margin-lg: 32px !default;
+
+    .k-pager-wrap {
+        .k-link,
+        .k-state-selected {
+            padding: 0;
```

Comment:
You can omit the rule altogether if the padding is exposed in theme-default. Since we are now using the monorepo, you can add a `$pager-buttons-padding` variable to theme-default (with a default value of the current padding), and then use it here, just like `$pager-buttons-size`.